### PR TITLE
Adapting various files with our project name and add committers

### PR DESCRIPTION
**Pull request by nhoening**
Also:
- add some helpful links
- add meeting notes of first TSC
- add current release checklist
- Try monthly meeting frequency.

Signed-off-by: Nicolas Höning <nicolas@seita.nl>

### CODE REVIEW

File: .github/workflows/create_agenda.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     - uses: pkgjs/meet
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        schedules: 2020-05-14T17:00:00.0Z/P4W
+        schedules: 2022-01-18T16:00:00.0Z/P1M
```
Comment (Flix6x) [inline .github/workflows/create_agenda.yml]: A period of 1 month should inevitably lead to weekend meetings. Probable better revert to an integer number of weeks.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,16 +17,27 @@ To contribute code to the project, first read over the [governance policies] pag
...
 
+## Development workflow
+
+At the moment, we have not established a workflow, other than we require contributions to be made in pull requests. Bigger topics should be discussed as [projects](https://github.com/FlexMeasures/flexmeasures/projects) first.
+
+
+## Coding style
+
+We program in Python and follow the PEP8 style guide.
+You can [read more](https://flexmeasures.readthedocs.io/en/latest/dev/introduction.html), e.g. on how to versioning works or how to apply our automatic code style tooling (Flake8, Black).
```
Comment (Flix6x) [inline CONTRIBUTING.md]: on how ~to~ versioning works

File: SECURITY.md [hunk 3]
```diff
@@ -2,20 +2,13 @@
...
 
 ## Reporting a Vulnerability
 
-Use this section to tell people how to report a vulnerability.
+Please report any vulnerabilities to nicolas@seita.nl.
 
-Tell them where to go, how often they can expect to get an update on a
-reported vulnerability, what to expect if the vulnerability is accepted or
-declined, etc.
+We value your expertise. Howver, currently we have no bug bounty program or similar.
```
Comment (Flix6x) [inline SECURITY.md]: How**e**ver

File: SUPPORT.md [hunk 4]
```diff
@@ -2,25 +2,25 @@ _Use this file to detail how to make get support forthe project. The structure b
 
 # Getting Help
 
-There are a few ways to connect with the PROJECT NAME project:
+There are a few ways to connect with the FlexMeasures project:
 
 * _provide email list details_
-* _provide Slack details_
-* _provide link to submit issue_
+* [Join the #FlexMEasures channel on LF Energy Slack](lfenergy.slack.com)
```
Comment (Flix6x) [inline SUPPORT.md:8]: Typo in FlexM E asures

File: SUPPORT.md [hunk 5]
```diff
@@ -2,25 +2,25 @@ _Use this file to detail how to make get support forthe project. The structure b
...
 ## How to Ask for Help
 
-If you have trouble installing, building, or using PROJECT NAME, but there's not yet reason to suspect you've encountered a genuine bug,
+If you have trouble installing, building, or using FlexMeasures, but there's not yet reason to suspect you've encountered a genuine bug,
 start by posting a question to the mailing list or Slack channel. This is the place for question such has "How do I...".
 
 ## How to report a bug or request an enhancement
 
-PROJECT NAME manages bug and enhancement using it's [issue tracker]. The issue template will guide you on making an effective report.
+FlexMeasures manages bug and enhancement using it's [issue tracker](https://github.com/FlexMeasures/flexmeasures/issues). The issue template will guide you on making an effective report.
```
Comment (Flix6x) [inline SUPPORT.md]: Typo: **its** (it seems also in the original template)

File: tsc/CHARTER.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 _Template - update with Adopted Date when TSC approves and send to operations@lfenergy.org to file_
 
-# Technical Charter (the "Charter") for [PROJECT NAME] a Series of LF Projects, LLC
+# Technical Charter (the "Charter") for [FlexMeasures] a Series of LF Projects, LLC
 
 **Adopted [DATE]**
 
-This charter (the "Charter") sets forth the responsibilities and procedures for technical contribution to, and oversight of, the [PROJECT NAME] project, which has been established as [PROJECT NAME] a Series of LF Projects, LLC (the "Project").  LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All Contributors to the Project must comply with the terms of this Charter.
+This charter (the "Charter") sets forth the responsibilities and procedures for technical contribution to, and oversight of, the [FlexMeasures] project, which has been established as [FlexMeasures] a Series of LF Projects, LLC (the "Project").  LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All Contributors to the Project must comply with the terms of this Charter.
```
Comment (Flix6x) [inline tsc/CHARTER.md]: I think the [ ] markers can go, too

File: tsc/meetings/2021-11-29.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+# FlexMEasures TSC Meeting - 2021-11-29 15:30 CET
```
Comment (Flix6x) [inline tsc/meetings/2021-11-29.md]: Typo in FlexM **E** asures

**Review by Flix6x — APPROVED** [review]
Just one serious issue with the meeting schedule. :)

**Review by jmertic — APPROVED** [review]

**Pull request merged**